recipes-kernel/kernel-modules/kernel-module-qca6564_git.bb

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2019 Digi International Inc.
+
+SUMMARY = "Qualcomm QCA6564 wireless driver module"
+LICENSE = "ISC"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/ISC;md5=f3b90e78ea0cffb20bf5cca7947a896d"
+
+# Reference Qualcomm tag/version
+PV = "v4.2.89.63"
+
+SRCBRANCH = "qca6564/master"
+SRCREV = "2e780b29209ca6b0f71442be5fd36cf0ddddc661"
+
+SRC_URI = "\
+    git://github.com/digi-embedded/qcacld-2.0.git;protocol=git;branch=${SRCBRANCH} \
+    file://modprobe-qca6564.conf \
+"
+
+LOCALVERSION ?= "-${SRCBRANCH}"
+
+S = "${WORKDIR}/git"
+
+inherit module
+
+EXTRA_OEMAKE += "CONFIG_LINUX_QCMBR=y WLAN_OPEN_SOURCE=1"
+# Explicity state it is not a QC platform, if not the driver will try to remap
+# memory that is not allowed in ARMv6
+EXTRA_OEMAKE += "CONFIG_NON_QC_PLATFORM=y"
+# Flag to compile the debug version (1 - enabled, rest of values - disabled)
+EXTRA_OEMAKE += "BUILD_DEBUG_VERSION=0"
+# Flags for SDIO interface
+EXTRA_OEMAKE += "CONFIG_CLD_HL_SDIO_CORE=y"
+
+do_compile_prepend() {
+    export BUILD_VER=${PV}
+}
+
+do_install_prepend() {
+    sed -i -e "s/gVhtTxMCS=2/gVhtTxMCS=0/g" ${WORKDIR}/git/firmware_bin/WCNSS_qcom_cfg.ini
```

Comment:
Do this change on D, so we allow it to be run every time.

Reply:
Done